src/compressed_tensors/offload/cache/disk.py

```
@@ -32,8 +32,8 @@ class DiskCache(OffloadCache):
 
     offload_device = "disk"
 
-    # offloaded tensors -> weight info
-    index: dict[torch.Tensor, dict[str, str]] = dict()
+    # id(offloaded tensor) -> weight info
+    index: dict[int, dict[str, str]] = dict()
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)